trade-finance-manager-ui/server/types/view-models/utilisation-report-reconciliation-for-report-view-model.ts

```diff
@@ -30,5 +30,6 @@ export type UtilisationReportReconciliationForReportViewModel = BaseViewModel &
   formattedReportPeriod: string;
   reportId: number;
   feeRecords: FeeRecordViewModelItem[];
-  errorSummary: [ErrorSummaryViewModel] | undefined;
+  addPaymentError: [ErrorSummaryViewModel] | undefined;
+  facilityIdQueryError: [ErrorSummaryViewModel] | undefined;
```

Comment:
Rather than make these arrays of only one item could we make them just e.g.: `addPaymentError: ErrorSummaryViewModel | undefined;`?
It made slightly more sense as `errorSummary` as that is an array in all other view models, so might have caused confusion if it were not an array but here I feel it is the other way around.